src/interfaces/idb.py

```diff
@@ -51,8 +50,7 @@ def put_items(
             database_name (str): _description_
             collection_name (str): _description_
         """
-        for item in items:
-            self.put_item(item, database_name, collection_name)
```

Comment:
why are you removing this implementation?

It's a default implementation that implementing class can use without having to implement it. If they need specific, bulk insert, they can simply override. Please, revert.

Also, if you consider removing my code, think about why I have it there in the first place. I have thought about all of this hard and long :) So if there is something somewhere, there is a reason for it. If in doubt, ask me.

Reply:
I made the switch because I was getting insertion errors when using the old implementation. I cannot remember the exact error. I will revert his back and see if the error comes up again. 

For this specific implementation i will override the method as you suggested.

Reply:
fixed.